### DISCUSSION TITLE
Cache tags removed from docs - going to be deprecated?

### DISCUSSION THREAD

**Discussion by atymic** [laravel/framework#48435] [Q&A]
In https://github.com/laravel/docs/commit/63dde39470588ab1ad39cea9a592f0838ca5ad47 cache tagging documentation was removed. Is there plans to depreciate the feature?

**Comment by sts-ryan-holton** [comment]
I noticed this too, what's the deal here?

**Comment by shaedrich** [comment]
From the comments:

> Taylor posted some explanation here: https://github.com/laravel/framework/pull/48078#issuecomment-1716332925
